contractcourt/commit_sweep_resolver_test.go

```diff
@@ -244,3 +246,92 @@ func TestCommitSweepResolverDelay(t *testing.T) {
 		t.Fatal("unexpected resolver report")
 	}
 }
+
+// TestCommitSweepResolverLocalBreach tests resolution when the local node
+// publishes a breached output (one that is swept by the remote party).
+func TestCommitSweepResolverLocalBreach(t *testing.T) {
```

Comment:
nb: you might want to consider making a table based sweep resolver test instead, to not duplicate most of `TestCommitSweepResolverDelay`.

Reply:
Done on the new commit.